modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/dynamic/data/mapping/util/JournalArticleImportDDMFormFieldValueTransformer.java

```diff
@@ -68,26 +69,40 @@ public void transform(DDMFormFieldValue ddmFormFieldValue)
 
 			JournalArticle journalArticle = null;
 
+			long originalArticlePrimaryKey = jsonObject.getLong(
+				"articlePrimaryKey");
+			long originalClassPK = jsonObject.getLong("classPK");
+
 			long articlePrimaryKey = GetterUtil.getLong(
 				_portletDataContext.getNewPrimaryKey(
 					JournalArticle.class + ".primaryKey",
-					jsonObject.getLong("articlePrimaryKey")));
+					originalArticlePrimaryKey));
 
 			if (articlePrimaryKey != 0) {
 				journalArticle =
 					_journalArticleLocalService.fetchJournalArticle(
 						articlePrimaryKey);
 			}
 
-			if (journalArticle == null) {
+			if ((journalArticle == null) &&
+				((originalArticlePrimaryKey != 0) || (originalClassPK != 0))) {
+
 				if (_log.isWarnEnabled()) {
 					_log.warn(
 						"Unable to get journal article with primary key " +
 							articlePrimaryKey);
 				}
 
-				_portletDataContext.removePrimaryKey(
-					ExportImportPathUtil.getModelPath(_stagedModel));
+				Map<String, String> postProcess =
+					(Map<String, String>)
+						_portletDataContext.getNewPrimaryKeysMap(
+							JournalArticle.class + ".postProcess");
```

Comment:
Can you rename `postProcess` to `postProcessArticlesUuids`

Reply:
renamed to `postProcessArticleUuids`